### PR TITLE
refactor(experimental): rename transaction errors

### PR DESCRIPTION
**Pull request by lorisleiva**
Renames the following transaction errors:

```ts
// Before.
export const SOLANA_ERROR__TRANSACTION_MISSING_SIGNATURES = 5663010 as const;
export const SOLANA_ERROR__TRANSACTION_SIGNATURE_NOT_COMPUTABLE = 5663011 as const;
export const SOLANA_ERROR__TRANSACTION_MISSING_ADDRESS = 5663012 as const;
export const SOLANA_ERROR__TRANSACTION_MISSING_FEE_PAYER = 5663013 as const;
export const SOLANA_ERROR__TRANSACTION_INVALID_NONCE_TRANSACTION_NO_INSTRUCTIONS = 5663014 as const;
export const SOLANA_ERROR__TRANSACTION_INVALID_NONCE_TRANSACTION_FIRST_INSTRUCTION_NOT_ADVANCE_NONCE = 5663015 as const;

// After.
export const SOLANA_ERROR__TRANSACTION_SIGNATURES_MISSING = 5663010 as const;
export const SOLANA_ERROR__TRANSACTION_ADDRESS_MISSING = 5663011 as const;
export const SOLANA_ERROR__TRANSACTION_FEE_PAYER_MISSING = 5663012 as const;
export const SOLANA_ERROR__TRANSACTION_FEE_PAYER_SIGNATURE_MISSING = 5663013 as const;
export const SOLANA_ERROR__TRANSACTION_INVALID_NONCE_TRANSACTION_INSTRUCTIONS_MISSING = 5663014 as const;
export const SOLANA_ERROR__TRANSACTION_INVALID_NONCE_TRANSACTION_FIRST_INSTRUCTION_MUST_BE_ADVANCE_NONCE = 5663015 as const;
```

### CODE REVIEW

**Comment by lorisleiva** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2261?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#2262** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2262?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2261** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2261?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#2260** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2260?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2259** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2259?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2257** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2257?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2256** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2256?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2255** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2255?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2254** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2254?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2253** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2253?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2252** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2252?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2251** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2251?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2250** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2250?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2249** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2249?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @lorisleiva and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/errors/src/codes.ts [hunk 1]
```diff
@@ -267,8 +268,8 @@ export const SOLANA_ERROR__INVARIANT_VIOLATION_SWITCH_MUST_BE_EXHAUSTIVE = 99000
  * is not safe, for a variety of reasons covered here: https://stackoverflow.com/a/28818850
  */
 export type SolanaErrorCode =
-    | typeof SOLANA_ERROR__TRANSACTION_MISSING_SIGNATURES
-    | typeof SOLANA_ERROR__TRANSACTION_SIGNATURE_NOT_COMPUTABLE
+    | typeof SOLANA_ERROR__TRANSACTION_SIGNATURES_MISSING
+    | typeof SOLANA_ERROR__TRANSACTION_FEE_PAYER_SIGNATURE_MISSING
```
Comment (buffalojoec) [inline packages/errors/src/codes.ts:272]: Just noticing this now, but this export list should probably be either alphabetical or in the same order as above.

**Review by buffalojoec — APPROVED** [review]

File: packages/errors/src/codes.ts (hunk 1 above)
Comment (buffalojoec) [reply packages/errors/src/codes.ts:272]: I _think_ this one does, but it also does some other annoying stuff, so yeah scoping it to here is probably best.
https://typescript-eslint.io/rules/sort-type-constituents/

Comment (steveluscher) [reply packages/errors/src/codes.ts:272]: Totally fine with adding such a sorter to `@solana/eslint-config-solana`.

**Review by steveluscher — APPROVED** [review]
Added via Giphy

**Comment by steveluscher** [comment]
### Merge activity

* **Mar 5, 10:18 AM PST**: @steveluscher started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2261).

**Pull request merged**